src/css_composer/view/CssRulesView.js

```diff
@@ -61,19 +62,50 @@ module.exports = require('backbone').View.extend({
       rendered = view.render().el;
     }
 
+    const mediaWidth = this.getMediaWidth(model.get('mediaText'));
+    const styleBlockId = `#${this.pfx}rules-${mediaWidth}`;
+
     if (rendered) {
-      if (fragment) fragment.appendChild(rendered);
-      else this.$el.append(rendered);
+      if (fragment) {
+        fragment.querySelector(styleBlockId).appendChild(rendered);
+      } else {
+        let $stylesContainer = this.$el.find(styleBlockId);
+        $stylesContainer.append(rendered);
+      }
     }
 
     return rendered;
   },
 
+  getMediaWidth(mediaText) {
+    return (
+      mediaText &&
+      mediaText.replace(`(${this.config.mediaCondition}: `, '').replace(')', '')
```

Comment:
mmm there is no `mediaCondition` inside `this.config`, you should get it from `this.em.getConfig('mediaCondition')`